pio-proc/src/lib.rs

```diff
@@ -136,7 +136,11 @@ impl syn::parse::Parse for PioFileMacroArgs {
                 let mut p = PathBuf::new();
 
                 if path.is_relative() {
-                    p.push(env!("CARGO_MANIFEST_DIR"));
+                    if let Ok(crate_dir) = std::env::var("CARGO_MANIFEST_DIR") {
```

Comment:
I think this is better as var_os. POSIX file paths do not have to be valid UTF-8 or representable as Unicode and as it's going straight into a PathBuf, might as well avoid the conversion.

Reply:
Fixed!